components/plugin_marketplace/marketplace_list/navigation_row/navigation_row.test.tsx

```diff
@@ -60,8 +58,8 @@ describe('components/marketplace/navigation_row', () => {
 
         wrapper.find('NavigationButton').simulate('click', {preventDefault: jest.fn});
 
-        expect(wrapper.instance().props.onNextPageButtonClick).toHaveBeenCalledTimes(0);
-        expect(wrapper.instance().props.onPreviousPageButtonClick).toHaveBeenCalledTimes(1);
+        expect((wrapper.instance() as NavigationRow).props.onNextPageButtonClick).toHaveBeenCalledTimes(0);
```

Comment:
You may be able to do this in order to avoid using `as` here:

```jsx
const wrapper = shallow<NavigationRow>(
    <NavigationRow {...props}/>,
);

Reply:
TIL That for the suggestion :+1: